src/controllers/MediaConnectorController.ts

```diff
@@ -158,3 +160,45 @@ export class MediaConnectorController {
         if (deprecatedType === DeprecatedMediaType.collection) return MediaType.collection;
     };
 }
+
+/**
+ * The ConnectorAuthenticationController is responsible for authenticating regarding media connectors.
+ * Methods inside this controller can be called by `window.SDK.mediaConnector.authentication.{method-name}`
+ */
+export class ConnectorAuthenticationController {
+    /**
+     * @ignore
+     */
+    #editorAPI: EditorAPI;
+
+    /**
+     * @ignore
+     */
+    constructor(editorAPI: EditorAPI) {
+        this.#editorAPI = editorAPI;
+    }
+
+    /**
+     * This method sets the token for the 'chili' authentication type.
```

Comment:
more info needed, what is this token, for what is it needed? What does it enable?